wtxmgr/db.go

```diff
@@ -1287,6 +1288,124 @@ func deleteRawUnminedInput(ns walletdb.ReadWriteBucket, outPointKey []byte,
 	return nil
 }
 
+// serializeLockedOutput serializes the value of a locked output.
+func serializeLockedOutput(id uint64, expiry time.Time) []byte {
+	var v [8 + 8]byte
+	byteOrder.PutUint64(v[:8], id)
+	byteOrder.PutUint64(v[8:], uint64(expiry.Unix()))
+	return v[:]
+}
+
+// deserializeLockedOutput deserializes the value of a locked output.
+func deserializeLockedOutput(v []byte) (uint64, time.Time) {
+	lockID := byteOrder.Uint64(v[:8])
+	expiry := time.Unix(int64(byteOrder.Uint64(v[8:])), 0)
+	return lockID, expiry
+}
+
+// isLockedOutput determines whether an output is locked. If it is, its assigned
+// ID is returned, along with its absolute expiration time. If the output lock
+// exists, but its expiration has been met, then it's removed.
```

Comment:
Comment here not super accurate in that it's the caller that will eventually remove them.